--- old/make/defs.make	2015-02-18 23:07:13.184453987 -0200
+++ new/make/defs.make	2015-02-18 23:07:13.114453987 -0200
@@ -330,6 +330,13 @@
   LIBARCH/arm     = arm
   LIBARCH/zero    = $(ZERO_LIBARCH)
 
+  # Override LIBARCH for ppc64le
+  ifeq ($(BUILDARCH), ppc64)
+    ifeq ($(OPENJDK_TARGET_CPU_ENDIAN), little)
+      LIBARCH = ppc64le
+    endif
+  endif
+
   LP64_ARCH = sparcv9 amd64 ia64 ppc64 zero
 endif
 
